2-mandatory/4-stocks.js

```diff
@@ -34,7 +34,21 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+    let average =[]
+    let sum = 0;
+    for (let i = 0; i < CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS.length; i++) {
+      for (let j = 0; j < CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS[i].length; j++) {
+        sum += CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS[i][j];
+      }
+      average.push(
+        (sum / CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS[i].length).toFixed(2) *
```

Comment:
As an alternative multiply the number by 100, then Math.round() it; then divide by 100

That way, you don't have to convert a number to a string , then multiply by 1 to convert it to a number again